package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@italia/spid-express",
-  "version": "2.0.5",
+  "version": "3.0.0",
```

Comment:
why this?

Reply:
because the previous version is not compatible with person data vault, 2.0.* implemented integration with User registry api at version 0.0.1 completely changed.

Reply:
I mean, we bump package version at release time, so we shouldn't do that in a PR. Anyway, fine for me